package.json

```diff
@@ -105,6 +108,7 @@
     "prettier": "^2.6.2",
     "prettier-plugin-packagejson": "^2.2.17",
     "rimraf": "^3.0.2",
+    "simple-git-hooks": "^2.8.0",
```

Comment:
this dep looks nice and i like that it can be opt-in and it's smaller than `husky`

Reply:
though i guess socket doesn't like the `postinstall`